src/js/post-911-gib-status/containers/StatusPage.jsx

```diff
@@ -10,10 +10,6 @@ import EnrollmentHistory from '../components/EnrollmentHistory';
 import UserInfoSection from '../components/UserInfoSection';
 
 class StatusPage extends React.Component {
-  componentDidMount() {
```

Comment:
Noticed a duplicate fetch of data here and it didn't seem necessary so removed it.